src/unix/fs.c

```diff
@@ -1240,9 +1241,28 @@ static ssize_t uv__fs_write(uv_fs_t* req) {
   return r;
 }
 
-static ssize_t uv__fs_copyfile(uv_fs_t* req) {
 #if defined(__APPLE__) && !TARGET_OS_IPHONE
-  /* On macOS, use the native copyfile(3). */
+
+static ssize_t uv__fs_clonefile_mac(uv_fs_t* req) {
+  static int has_clone_acl = 1;
+
+  /* Note: clonefile() does not set the group ID on the destination
+   * file correctly. */
+  if (!clonefile(req->path, req->new_path, has_clone_acl ? UV__FS_CLONE_ACL : 0))
```

Comment:
This is just over the 80 column limit. Try to avoid using a ternary inside a call statement.